EZSwiftExtensionsTests/DateTests.swift

```diff
@@ -40,17 +40,26 @@ class DateTests: XCTestCase {
         // Given
         let expectedResult = Date(timeIntervalSince1970: 784_111_777)
         let fromGMT = TimeInterval(NSTimeZone.local.secondsFromGMT())
+        let fromStartOfDay = TimeInterval(8 * 3600 + 49 * 60 + 37) // seconds from start of day
 
         // When
         let rfc1123 = Date(httpDateString: "Sun, 06 Nov 1994 08:49:37 GMT")
         let rfc850 = Date(httpDateString: "Sunday, 06-Nov-94 08:49:37 GMT")
         let asctime = Date(httpDateString: "Sun Nov  6 08:49:37 1994")
+        let iso8601DateOnly = Date(httpDateString: "1994-11-06")
```

Comment:
I made some new commits. I am using the date time of the first commit of this pod as the base date just for this test scenario.

I won't be using the `string` attribute of this test class as the base date for this test scenario because it is being used by other test cases, which are actually failing in my case due to my time zone.

I believe a rewrite of all the tests to improve their robustness is required and maybe we should do it in another PR?

I also intend to follow up with another PR to take in the locale as a parameter for various date/string related functions. Maybe I can do them together in that PR?